modules/tests/shared/src/test/scala/lucuma/core/math/WavelengthSuite.scala

```diff
@@ -6,44 +6,61 @@ package lucuma.core.math
 import cats.Eq
 import cats.Order
 import cats.Show
-import cats.kernel.laws.discipline._
-import cats.syntax.all._
+import cats.kernel.laws.discipline.*
+import cats.syntax.all.*
 import coulomb.*
-import coulomb.policy.spire.standard.given
 import coulomb.syntax.*
-import eu.timepit.refined.auto._
-import eu.timepit.refined.cats._
-import eu.timepit.refined.numeric._
-import eu.timepit.refined.scalacheck.numeric._
+import eu.timepit.refined.auto.*
+import eu.timepit.refined.cats.*
+import eu.timepit.refined.numeric.*
+import eu.timepit.refined.scalacheck.numeric.*
+import eu.timepit.refined.types.numeric.PosBigDecimal
 import eu.timepit.refined.types.numeric.PosInt
-import lucuma.core.math.arb._
+import lucuma.core.math.arb.*
 import lucuma.core.math.units.{_, given}
 import lucuma.core.optics.Format
 import lucuma.core.optics.laws.discipline.FormatTests
-import monocle.law.discipline._
+import monocle.law.discipline.*
+import org.scalacheck.Arbitrary
+import org.scalacheck.Arbitrary.arbitrary
 import org.scalacheck.Prop
-import org.scalacheck.Prop._
+import org.scalacheck.Prop.*
 import spire.math.Rational
 
 import java.math.RoundingMode
+import scala.language.implicitConversions
 
 final class WavelengthSuite extends munit.DisciplineSuite {
-  import ArbWavelength._
+  //  import ArbQuantity._   Why won't this work?
```

Comment:
commented line?

Reply:
Thanks, removed.  I'm still unsure why it didn't work but content to let the mystery be.

Reply:
I think I've seen that before where you need to import each member and the wildcard doesn't work